### DISCUSSION TITLE
Add validation of environment variables param in container-app module

### DISCUSSION THREAD

**Discussion by raix** [platformplatform/PlatformPlatform#641] [Ideas]
### Detailed description?

**Is your feature request related to a problem? Please describe.**
When setting environment variables using the `container-app.bicep` module only values of type "string" should be allowed

If passing on a number value in environment variables it will result in:
![Image](https://github.com/user-attachments/assets/6706412a-5686-4c1c-ac7c-3a950da82385)

**Describe the solution you'd like**
Ideally the param validation should happen before deployment

**Describe alternatives you've considered**
Currently setting string values

**Additional context**

Example of a solution adding types in params:
![Image](https://github.com/user-attachments/assets/3333cf11-e13b-4d59-b5ec-337e7abea5c8)

### Feature Type

Infrastructure

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by tjementum** [comment]
For now, I converted this to a discussion. I'm trying to keep the issues clean so they do not contain a lot of items that are just ideas and will not be done in the short run.
